### DISCUSSION TITLE
Download failed. Opening ticket as requested.

### DISCUSSION THREAD

**Discussion by therionveen** [aclap-dev/video-downloadhelper#585] [Help and Q&A]
### Your question

Windows 10 Pro 64 22h2 (19045.4412)
MS Edge (Version 125.0.2535.51 (Official build) (64-bit)

Bug details:

{
  "version": "9.0.1.18",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://clubepbe.alumy.com/playlists/monitoramento-de-progresso-e-mensuracao-de-resultados/posts/introducao-5",
  "error": {
    "details": "SideDownload error: : Not detecting m3u8/hls with non standard extension and non standard mime type\r\n    Last message repeated 4 times\r\n[in#0 @ 00000233e72aa500] Error opening input: Invalid data found when processing input\r\nError opening input file https://subplaylist-videozz.k8s.eduzz.com/?t=eyJhbGciOiJIUzI1NiJ9.eyJrZXkiOiJhZjBmNjZkMy01ODU5LTQzMmUtODA1ZS03MWU5MzQzNDMzOGIiLCJzZXNzaW9uS2V5IjoiYmxqcjV1dXAiLCJwcm9kdWNlcktleSI6IjY0ODY5MDIxIiwicHJvZHVjZXJFbWFpbCI6ImNvbnRhdG9AaW5wYmUuY29tLmJyIiwiY29uc3VtZXJFbWFpbCI6ImthdHlfc3RyZWNrQGhvdG1haWwuY29tIiwiaW1wcmludCI6dHJ1ZSwic3RvcmFnZSI6InIyIiwicGwiOiI0ODBwX3BsYXlsaXN0Lm0zdTgiLCJpYXQiOjE3MTY0OTY3NzAsImV4cCI6MTcxNjY2OTU3MH0.rgNF0ZN9cJhi2HaMf6XGht7MAJR22q_ZbTNBY3HkdEk.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2274018351552885",
    "error": "coapp_failure",
    "id": "coapp_failure_94ffd9a9-d6f7-4d9d-9d4a-58b2ffb628e9"
  }
}

### Addon & CoApp version and details

VDH Version 9.0.1.18

CoApp
 "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "E:\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\*",
  "converterBinary": "E:\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"

I have no idea if Edge is detecting the coapp but VDH stopped asking for it to be installed after I did it. Also have no idea how (and where) I should run "vdhcoapp install" in case I need to.

**Comment by paulrouget** [comment]
Your coapp is properly installed.

It's a type of video we don't support yet. Working on it.
